.github/workflows/actionlint.yml

```diff
@@ -1,16 +1,7 @@
 # This file is synced from the `.github` repository, do not modify it directly.
 name: Actionlint
 
-on:
-  push:
-    branches:
-      - main
-      - master
-    paths:
-      - '.github/workflows/*.ya?ml'
-  pull_request:
-    paths:
-      - '.github/workflows/*.ya?ml'
+on: [push, pull_request]
```

Comment:
We can probably keep `push` on `main/master` to avoid duplicating checks in PRs. The PR checks would show both push & pull_request otherwise.